app/assets/javascripts/crosswalks.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
@mphprogrammer I want to verify that you really want a crosswalks.js and crosswalks.scss file?